packages/eslint-config/shared.js

```diff
@@ -154,7 +154,8 @@ module.exports = {
         // 'recommended' rules we alter
         '@typescript-eslint/no-explicit-any': 'warn',
         '@typescript-eslint/no-var-requires': 'off',
-        '@typescript-eslint/ban-types': 'warn',
+        '@typescript-eslint/no-require-imports': 'off',
```

Comment:
The `no-var-requires` above is now deprecated in favour of this rule. I have left the original in for the moment and matched it's level.